stream.js

```diff
@@ -139,14 +139,14 @@ Stream.prototype._resume = function () {
     return
   }
 
+  if (this.state === STREAM_STATE.INITIALIZING)
+    return // not ready yet
+
   if (!this.sink || this.sink.paused) {
     this.state = STREAM_STATE.PAUSED
     return
   }
 
-  if (this.state === STREAM_STATE.INITIALIZING)
-    return // not ready yet
```

Comment:
before what could happen is that you would trigger a resume and the sink was either no available or pasued. In this case we end up in state PAUSED. If you then resume again and sink is not paused, you will be allowed to continue and because the db is empty, you'll end up in the BLOCK_STATE.END_OF_STREAM where for a live stream we set skip_next true because we assume that you already got whatever the latest data was.